app/src/services/object.js

```diff
@@ -76,6 +76,21 @@ const service = {
     }
   },
 
+  /**
+   * @function searchMetadata
+    * Search and filter for specific metadata keys
+   * @param {object} [params.metadata] Optional object of metadata keys to filter on
+   * @returns {Promise<object[]>} The result of running the find operation
+   */
+  searchMetadata: (params) => {
+    return Metadata.query()
+      .modify('filterKey', { metadata: params.metadata })
```

Comment:
minor point, it might simplify things if you just pass the params.metadata array, instead of nesting it in another object.

Reply:
Will leave as is for now. It follows the same patterns as searchObjects